pkg/koordlet/metrics/cpu_burst.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	ContainerCFSBurstCPU = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "container_cfs_burst_cpu_cores",
+		Help:      "Number of cores used by cfs burst, in container level",
+	}, []string{NodeKey, PodNamespace, PodName, ContainerID, ContainerName})
+
+	ContainerCFSBurstQuota = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "container_cfs_burst_quota",
```

Comment:
This metric records the value of container cgroup's scaled `cpu.cfs_quota_us`. But the naming `container_cfs_burst_quota` may be confused with `cpu.cfs_burst_us`.